packages/salesforcedx-vscode-apex/src/commands/forceApexExecute.ts

```diff
@@ -260,13 +259,21 @@ export class ApexLibraryExecuteExecutor extends LibraryCommandletExecutor<
   }
 }
 
-export async function forceApexExecute(
-  isDebugging: boolean
-) {
+export async function forceAnonApexExecute() {
```

Comment:
How does the team feel about using the syntax: 
```
export const forceAnonApexExecute = async () => { 
```
as opposed to named functions?

Reply:
Let's hold off for now.  I know there's good reason to use arrow functions when using an inlined function, but not clear on the benefits when exporting.  This reply on SO says there isn't a benefit for exported functions.  https://stackoverflow.com/questions/52425643/node-js-export-a-normal-function-or-arrow-function.  Since this codebase exports named functions, I suggest a work item be logged, look into any benefits, and then, update the codebase if we go that way.  Changing it just here, I feel it would be "swimming up against the current"